src/test/java/gov/healthit/chpl/aqa/stepDefinitions/ChplAPISteps.java

```diff
@@ -24,4 +33,46 @@ public void verifyContentUnderCertifiedHealthITProductListingAndURL(final String
         assertEquals(actualText, text);
         assertEquals(actualURL, urlLink);
     }
+
+    /**
+     * Assert that title should be CHPL API.
+     * @param title expected as CHPL API
+     */
+    @Then("^CHPL API page title should be \"([^\"]*)\"$")
+    public void chplApiPageTitleShouldBe(final String title) {
+        String chplAPITitle = getDriver().getTitle();
+        Assert.assertEquals(chplAPITitle, title);
+    }
+
+    /**
+     * Click all the controller elements link.
+     * @param controllerName as the name of Controller
+     */
+    @When("^I click on \"([^\"]*)\" link$")
+    public void iClickOnLink(final String controllerName) {
+        WebElement link = ChplAPIPage.controllerLink(getDriver(), controllerName);
+        ((JavascriptExecutor) getDriver()).executeScript("arguments[0].click();", link);
+    }
+
+    /**
+     * Assert that APIs from webpage is present in the example API Endpoints.
+     * @param apiEndpoints as the details of controller
+     */
+    @Then("^list of \"([^\"]*)\" operations should be displayed without deprecated calls$")
+    public void listOfOperationsShouldBeDisplayed(final String apiEndpoints) {
+        WebDriver driver = getDriver();
+        // Get element from the webpage into the list
+        List<WebElement> listWebElements = ChplAPIPage.controllerElementList(driver);
+        // Creating a list to hold all the API listed on the webpage by swagger
+        List<String> apiListFromPage = new ArrayList<>();
+        // Get text of the Web Elements list
+        for (WebElement wElement : listWebElements) {
+            apiListFromPage.add(wElement.getText());
+        }
+        System.out.println("apiEndpoints" + apiEndpoints);
+        // Checking if the API from the webpage is present in the example API endpoints
+        for (String api : apiListFromPage) {
+            assertTrue(apiEndpoints.contains(api));
```

Comment:
This is an excellent change. The only thing I see now is that  your check for the API call on the page to be in the Examples data could fail if someone edited an API call and removed words from the end. For example, if the page had the API call "Create a new trigger" that would be found in the string related to the scheduler-controller, even though it's not the complete API call as defined in that data. Would recommend taking the apiEndpoints value and splitting it into an array or List, then using some List.contains(api) call to check to see that it matches exactly.

Also consider using something other than a comma for delineating the API calls; it's possible for an API description to have a comma in it. Could probably use multiple characters if that'd help. I was going to suggest a pipe, but that doesn't work with the feature definition. Maybe `*`, or `--`, or even something silly like `xxx`